BREAKING_CHANGES.md

```diff
@@ -1,5 +1,13 @@
 # Breaking Changes
 
+## 2.0.0
```

Comment:
Should this be 1.0.0 or 2? Did not mention it for other packages but just thought about it.

Reply:
1.x.x already exist.
The breaking changes is missing info about 1.0.0
https://www.nuget.org/packages/Chinook.DynamicMvvm/1.2.2-feature.Uno5Update.2